From 49fb63bac72c3ba8f6b957180a8b550273f492aa Mon Sep 17 00:00:00 2001
From: Elaine Zhang <zhangqing@rock-chips.com>
Date: Wed, 11 Sep 2019 09:21:45 +0800
Subject: [PATCH] clk: rockchip: rk3399: fix up the FRAC_MAX_PRATE limit for
 i2s and uart

I2S and UART may be application requirements higher than 30M frequency,
So increase the input frequency limit of the frac divider.
IC back-end emulation 800M is safety.

Change-Id: Ida63505b5124799ad8a64d70af4403eae121cbb8
Signed-off-by: Elaine Zhang <zhangqing@rock-chips.com>
---
 drivers/clk/rockchip/clk-rk3399.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/clk/rockchip/clk-rk3399.c b/drivers/clk/rockchip/clk-rk3399.c
index 5a649c3e680f..306909609cb3 100644
--- a/drivers/clk/rockchip/clk-rk3399.c
+++ b/drivers/clk/rockchip/clk-rk3399.c
@@ -21,8 +21,8 @@
 #include <dt-bindings/clock/rk3399-cru.h>
 #include "clk.h"
 
-#define RK3399_I2S_FRAC_MAX_PRATE       600000000
-#define RK3399_UART_FRAC_MAX_PRATE	600000000
+#define RK3399_I2S_FRAC_MAX_PRATE       800000000
+#define RK3399_UART_FRAC_MAX_PRATE	800000000
 #define RK3399_SPDIF_FRAC_MAX_PRATE	600000000
 #define RK3399_VOP_FRAC_MAX_PRATE	600000000
 #define RK3399_WIFI_FRAC_MAX_PRATE	600000000
-- 
2.35.3

